src/main/java/org/example/calculator/model/Calculator.java

```diff
@@ -0,0 +1,47 @@
+package org.example.calculator.model;
+
+import java.util.Arrays;
+import java.util.function.Function;
+
+public class Calculator {
+
+    private static final String DEFAULT_DELIMITER = ",|:";
+    private static final String CUSTOM_DELIMITER_REGEX = "//(.)\n";
+    private static final String CUSTOM_OPERANDS_REGEX = "//.\n(.+)";
+    private final String string;
+
+    public Calculator(String string) {
+        this.string = string == null ? "" : string;
+    }
+
+    public int calculate() {
+        String delimiter = CalculatorStringUtils.extractDelimiterWithRegex(string, CUSTOM_DELIMITER_REGEX, DEFAULT_DELIMITER);
+        String operandString = CalculatorStringUtils.extractOperandStringWithRegex(string, CUSTOM_OPERANDS_REGEX);
+        String[] operands = splitOperandStringWithDelimiter(operandString, delimiter);
```

Comment:
단순히 String의 함수를 호출하는 간단한 로직인데 이를 메서드로 추출하는 것은 조금 과하다고 생각되네요. 🤔

```suggestion
        String[] operands = operandString.split(delimiter);
```